.github/workflows/build.yml

```diff
@@ -249,6 +249,15 @@ jobs:
         with:
           fetch-depth: 0
           submodules: recursive
+      - name: Set up or Update vcpkg
+        run: |
+          if (!(Test-Path C:/vcpkg)) {
+            git clone https://github.com/microsoft/vcpkg.git C:/vcpkg
```

Comment:
vcpkg comes preinstalled on the `windows-2019` runner image (and all other GitHub actions runner images), and the location of it is specified by the environment variable `VCPKG_INSTALLATION_ROOT`, so you might be able to simplify the entire script to:

```pwsh
cd ${VCPKG_INSTALLATION_ROOT}
git pull
.\bootstrap-vcpkg.bat
```

Reply:
Thanks for the suggestion. Applied it.

Reply:
Applying this broke it again.

Reply:
Maybe it needs to be
```pwsh
cd $env:VCPKG_INSTALLATION_ROOT
```

Reply:
Looks like that did the trick